airbyte-server/src/main/java/io/airbyte/server/apis/AttemptApiController.java

```diff
@@ -6,11 +6,12 @@
 
 import io.airbyte.api.generated.AttemptApi;
 import io.airbyte.api.model.generated.InternalOperationResult;
+import io.airbyte.api.model.generated.SaveStatsRequestBody;
 import io.airbyte.api.model.generated.SetWorkflowInAttemptRequestBody;
 import io.airbyte.server.handlers.AttemptHandler;
 import javax.ws.rs.Path;
 
-@Path("/v1/attempt/set_workflow_in_attempt")
+@Path("/v1/attempt/")
```

Comment:
I've made this a less restrictive path so this will continue to override 2 different routes. 

This works since the path, while less restrictive, is still more restrictive than the general V1 path in the ConfigurationApi class.